modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -0,0 +1,827 @@
+    'databases_server'          => '',                                                                                # host for general output dbs
+    'dna_db_server'             => '',                                                                                # host for dna db
+    'pipe_db_port'              => '',                                                                                # port for pipeline host
+    'databases_port'            => '',                                                                                # port for general output db host
+    'dna_db_port'               => '',                                                                                # port for dna db host
+    'registry_host'             => '',                                                                                # host for registry db
+    'registry_port'             => '',                                                                                # port for registry db
+    'registry_db'               => '',                                                                                # name for registry db
+    'rnaseq_summary_file'       => '' || catfile( $self->o('rnaseq_dir'), $self->o('species_name') . '.csv' ),        # Set this if you have a pre-existing cvs file with the expected columns
+    'rnaseq_summary_file_genus' => '' || catfile( $self->o('rnaseq_dir'), $self->o('species_name') . '_gen.csv' ),    # Set this if you have a pre-existing genus level cvs file with the expected columns
+    'release_number' => '' || $self->o('ensembl_release'),
+    'species_name'        => '',                                                                                      # e.g. mus_musculus
+    'production_name'     => '',                                                                                      # usually the same as species name but currently needs to be a unique entry for the production db, used in all core-like db names
+    'taxon_id'            => '',                                                                                      # should be in the assembly report file
+    'uniprot_set'         => '',                                                                                      # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    'output_path'         => '',                                                                                      # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'assembly_name'       => '',                                                                                      # Name (as it appears in the assembly report file)
+    'registry_file'       => '' || catfile( $self->o('output_path'), "Databases.pm" ),                                # Path to databse registry for LastaZ and Production sync
```

Comment:
This is not needed in this config

Reply:
I guess it only refers to the last line. Removed registry_file.